templates/resources/consul_cluster.md.tmpl

```diff
@@ -14,6 +14,7 @@ description: |-
 {{ tffile "examples/resources/hcp_consul_cluster/resource.tf" }}
 
 -> **Note:** The `primary_link` attribute is related to federation, a feature that is currently in private beta.
+-> **Note:** The `plus` tier is currently in private beta.
```

Comment:
Just double-checked this with [the doc preview tool](https://registry.terraform.io/tools/doc-preview) and looks like there needs to be a blank line between the notes for them to render separately.

Also, could you move the banners below `# {{.Type}} ({{.Name}})`? I just realized we don't have a consistent pattern across our pages. 😔